blosc/blosc2.h

```diff
@@ -154,9 +155,19 @@ enum {
   BLOSC_SNAPPY = 3,
   BLOSC_ZLIB = 4,
   BLOSC_ZSTD = 5,
-  BLOSC_MAX_CODECS = 7,  //!< maximum number of reserved codecs
+  BLOSC_UDCODEC = 6,
+  BLOSC_LAST_CODEC = 7,  //!< sentinel
+  BLOSC_LAST_REGISTERED_CODEC = 32,  //!< sentinel
```

Comment:
I'd redefine this as:

```
BLOSC_LAST_REGISTERED_CODEC = BLOSC2_BDEFINED_CODECS + 0
```

and add a comment explaining that this number (0) should be increased as we register new codecs.